src/engine/framework/System.cpp

```diff
@@ -355,6 +392,144 @@ static void CloseSingletonSocket()
 #endif
 }
 
+#if defined(DAEMON_USE_FLOAT_EXCEPTIONS_AVAILABLE) && defined(__APPLE__)
+static void FloatingPointExceptionSignalHandler(int /* sig */, siginfo_t *sip, void* /* scp */)
```

Comment:
I think it's better to remove the sigaction call. We already register handlers for these signals in the standard startup code so this steps on them. I tested without the handler and it works as expected: the previously registered handler is called (or it crashes directly if `-nocrashhandler` is used).

Reply:
Done.